[2021-12-05 16:14:21,148] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_chapecoense 2021-11-29T09:00:00+00:00 [queued]>
[2021-12-05 16:14:21,153] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_chapecoense 2021-11-29T09:00:00+00:00 [queued]>
[2021-12-05 16:14:21,153] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2021-12-05 16:14:21,153] {taskinstance.py:881} INFO - Starting attempt 4 of 4
[2021-12-05 16:14:21,153] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2021-12-05 16:14:21,160] {taskinstance.py:901} INFO - Executing <Task(TwitterOperator): twitter_chapecoense> on 2021-11-29T09:00:00+00:00
[2021-12-05 16:14:21,161] {standard_task_runner.py:54} INFO - Started process 3801 to run task
[2021-12-05 16:14:21,212] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'twitter_dag', 'twitter_chapecoense', '2021-11-29T09:00:00+00:00', '--job_id', '127', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/twitter_dag.py', '--cfg_path', '/tmp/tmpw0hs_2gn']
[2021-12-05 16:14:21,212] {standard_task_runner.py:78} INFO - Job 127: Subtask twitter_chapecoense
[2021-12-05 16:14:21,230] {logging_mixin.py:112} INFO - Running <TaskInstance: twitter_dag.twitter_chapecoense 2021-11-29T09:00:00+00:00 [running]> on host kauai-VirtualBox
[2021-12-05 16:14:21,267] {base_hook.py:89} INFO - Using connection to: id: twitter_default. Host: https://api.twitter.com, Port: None, Schema: None, Login: None, Password: None, extra: XXXXXXXX
[2021-12-05 16:14:21,267] {taskinstance.py:1150} ERROR - name 'time' is not defined
Traceback (most recent call last):
  File "/home/kauai/datapipeline/venv/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 984, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/home/kauai/airflow/plugins/operators/twitter_operator.py", line 49, in execute
    for pg in hook.run():
  File "/home/kauai/airflow/plugins/hooks/twitter_hook.py", line 57, in run
    yield from self.paginate(url, session)
  File "/home/kauai/airflow/plugins/hooks/twitter_hook.py", line 46, in paginate
    data = self.connect_to_endpoint(full_url, session)
  File "/home/kauai/airflow/plugins/hooks/twitter_hook.py", line 34, in connect_to_endpoint
    time.sleep(5)
NameError: name 'time' is not defined
[2021-12-05 16:14:21,268] {taskinstance.py:1187} INFO - Marking task as FAILED. dag_id=twitter_dag, task_id=twitter_chapecoense, execution_date=20211129T090000, start_date=20211205T191421, end_date=20211205T191421
[2021-12-05 16:14:26,140] {local_task_job.py:102} INFO - Task exited with return code 1
